exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,38 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to the [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `timeout` (optional): Timeout in seconds used for HTTP request
+- `method` (optional): HTTP method, one of: "POST" or "PUT". "POST" is used by default if unset.
+- `compress` (optional): Option to enable compression (default false)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
+- `log_format` (optional): Format to post logs into Sumo. (default `json`)
+  - `text` - Logs will appear in SumoLogic in text format (taken from the field specified in log_key)
+  - `json` - Logs will appear in SumoLogic in json format.
+- `metric_format` (optional): The format of metrics you will be sending, either graphite or carbon2 or prometheus (Default is graphite).
```

Comment:
Any reason why we default to graphite or that's part of the boilerplate?

Reply:
I don't know. Is there any reason why `graphite` is default for https://github.com/SumoLogic/fluentd-output-sumologic ?
cc: @frankreno